tests/loggers/test_tensorboard.py

```diff
@@ -34,17 +35,26 @@ def test_tensorboard_hparams_reload(tmpdir):
 
     # verify artifacts
     assert len(os.listdir(os.path.join(folder_path, "checkpoints"))) == 1
-    #
-    # # verify tb logs
-    # event_acc = EventAccumulator(folder_path)
-    # event_acc.Reload()
-    #
-    # hparams_data = b'\x12\x84\x01"\x0b\n\tdrop_prob"\x0c\n\nbatch_size"\r\n\x0bin_features"' \
-    #                b'\x0f\n\rlearning_rate"\x10\n\x0eoptimizer_name"\x0b\n\tdata_root"\x0e\n' \
-    #                b'\x0cout_features"\x0c\n\nhidden_dim"\x04\n\x02b1"\x04\n\x02b2'
-    #
-    # assert event_acc.summary_metadata['_hparams_/experiment'].plugin_data.plugin_name == 'hparams'
-    # assert event_acc.summary_metadata['_hparams_/experiment'].plugin_data.content == hparams_data
+    
+    # verify tb logs
+    event_acc = EventAccumulator(folder_path)
+    event_acc.Reload()
+    
+    hparams_data = b'\x12\x93\x01"' \
+                   b'\x0b\n\tdrop_prob"' \
+                   b'\x0c\n\nbatch_size"' \
+                   b'\r\n\x0bin_features"' \
+                   b'\x0f\n\rlearning_rate"' \
+                   b'\x10\n\x0eoptimizer_name"' \
+                   b'\x0b\n\tdata_root"' \
+                   b'\x0e\n\x0cout_features"' \
+                   b'\x0c\n\nhidden_dim"' \
```

Comment:
seems some of these special characters change from one version to the other. perhaps you need to go for a regex matching instead of full equality check. not sure.

Reply:
yes, not sure why...